src/absil/ilreflect.fs

```diff
@@ -1942,7 +1941,7 @@ let createTypeRef (visited : Dictionary<_, _>, created : Dictionary<_, _>) emEnv
                 traverseType CollectTypes.All cx
 
         if verbose2 then dprintf "buildTypeDefPass4: Doing method constraints of %s\n" tdef.Name
-        for md in tdef.Methods.AsList do
+        for md in tdef.Methods.AsArray do
```

Comment:
I assume it will be fine if you remove the AsArray as well and just iterate the original collection

Reply:
the type is declared like ILAttributes